applications_dev/knowledgebase/mysql-knowledgebase.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: mysql-knowledgebase
+  namespace: argocd
+  labels:
+    purpose: knowledgebase
+spec:
+  destination:
+    namespace: knowledgebase
+    server: https://kubernetes.default.svc
+  project: default
+  source:
+    path: ionos_dev/knowledgebase/mysql
+    repoURL: https://github.com/BazRoe/dome-gitops.git
```

Comment:
@BazRoe wrong repoURL and targetRevision. should be https://github.com/DOME-Marketplace/dome-gitops.git and HEAD

Reply:
Thank you, i've updated it